snuba/admin/clickhouse/system_queries.py

```diff
@@ -288,12 +308,22 @@ def run_system_query_on_host_with_sql(
             raise UnauthorizedForSudo()
 
     validate_query(
-        clickhouse_host, clickhouse_port, storage_name, system_query_sql, sudo_mode
+        clickhouse_host,
+        clickhouse_port,
+        storage_name,
+        system_query_sql,
+        sudo_mode,
+        clusterless_mode,
```

Comment:
we are probably at the point where we should encode this in some kind of parameter structure so that changes like this have smaller blast radius, but I don't want to block this change on that